src/strategies/ocean-marketplace/index.ts

```diff
@@ -89,26 +84,42 @@ export async function strategy(
     result.pools.forEach((pool) => {
       if (pool.holderCount > 0 && pool.active) {
         pool.shares.map((share) => {
-          const userAddress = getAddress(share.userAddress.id).toLowerCase();
+          const userAddress = getAddress(share.userAddress.id);
           if (!userAddresses.includes(userAddress))
             userAddresses.push(userAddress);
           if (!score[userAddress]) score[userAddress] = BigNumber.from(0);
           let userShare =
             share.balance * (pool.oceanReserve / pool.totalShares);
           if (userShare > 0.0001) {
             score[userAddress] = score[userAddress].add(
-              bdToBn(userShare.toString(), options.decimals)
+              bdToBn(userShare.toString(), OCEAN_ERC20_DECIMALS)
             );
           }
         });
       }
     });
 
     userAddresses.forEach((address) => {
-      let parsedSum = parseFloat(formatUnits(score[address], options.decimals));
+      let parsedSum = parseFloat(formatUnits(score[address], OCEAN_ERC20_DECIMALS));
       return_score[address] = parsedSum;
     });
   }
 
+  // From graphUtils.ts => verifyResults => Scores need to match expectedResults.
+  // expectedResults should reflect broad token distributions @ blockHeight
+  // npm run test --strategy=ocean-marketplace | grep -E 'SUCCESS|ERROR'
+  if (options.expectedResults) {
+    let results = {}
+    Object.keys(options.expectedResults.scores).forEach(function(key) {
+      results[key] = return_score[key];
+    });
+
+    verifyResults(
+      JSON.stringify(results),
+      JSON.stringify(options.expectedResults.scores),
+      'Scores'
+    );
+  }
+
   return return_score || {};
```

Comment:
It seem the strategy return all the token holders when i test, something is wrong, you should only return score from voters, i think you need to change here
```suggestion
  return results || {};
```

Reply:
The strategy looks inside tokens inside the 1000 largest pools, and everything was being returned.

I have updated the strategy() function to only return the scores from the addresses provided/expected.